src/openrct2/platform/Platform.Linux.cpp

```diff
@@ -150,12 +155,12 @@ namespace Platform
         {
             log_fatal("failed to read /proc/self/exe");
         }
-#elif defined(__FreeBSD__)
-        const sint32 mib[] = { CTL_KERN, KERN_PROC, KERN_PROC_PATHNAME, -1 };
-        auto exeLen = sizeof(exePath);
-        if (sysctl(mib, 4, exePath, &exeLen, nullptr, 0) == -1)
-        {
-            log_fatal("failed to get process path");
+#elif defined(__FreeBSD__)		 +#elif defined(__OpenBSD__) || defined(__FreeBSD__)
```

Comment:
That doesn't look quite right

Reply:
Yeah, I noticed it right after sending it.